src/test/spec/retryable_writes/mod.rs

```diff
@@ -314,3 +314,45 @@ async fn mmapv1_error_raised() {
         e => panic!("expected command error, got: {:?}", e),
     }
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn label_not_added_first_read_error() {
+    let _guard = LOCK.run_exclusively().await;
+    assert!(label_not_added(false, 1).await);
+}
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn label_not_added_second_read_error() {
+    let _guard = LOCK.run_exclusively().await;
+    assert!(label_not_added(true, 2).await);
+}
+
+#[function_name::named]
+async fn label_not_added(retry_reads: bool, times: i32) -> bool {
+    let options = ClientOptions::builder().retry_reads(retry_reads).build();
```

Comment:
I think retryable writes should be enabled for these tests too? Otherwise, `self.inner.options.retry_writes != Some(false) && op.retryability() == Retryability::Write` will be false due to the first check, and we won't be testing whether labels are added to reads when retryable writes are enabled.

Reply:
Ah, right, I forgot they're enabled by default